.storybook/themes/news.scss

```diff
@@ -1 +1,188 @@
+				),
+				right-rail-main: (
+					gap: var(--global-spacing-6),
+					max-width: calc(var(--content-max-width) * 1px),
+					width: var(--content-scale-width),
+					margin: auto,
+				),
+				right-rail-full-width-2: (
+					margin: var(--global-spacing-6) 0 0 0,
+				),
+				right-rail-rail-container: (
+					grid-template-columns: initial,
+					grid-template-rows: 1fr,
+					gap: var(--global-spacing-6),
+				),
+				right-rail-main-side-rail: (
+					padding: initial,
+				),
+				right-rail-main-interior-item: (
+					padding: initial,
+					border-right: initial,
+				),
+				right-rail-footer: (
+					margin: var(--global-spacing-10) 0 var(--global-spacing-8) 0,
+				),
+				right-rail-advanced: (
+					gap: var(--global-spacing-6),
+				),
+				right-rail-advanced-navigation: (
+					position: sticky,
+					top: 0,
+					width: 100%,
+					z-index: 9,
+					margin: 0 0 var(--global-spacing-5) 0,
+				),
+				right-rail-advanced-full-width-1: (
+					margin: 0 0 var(--global-spacing-5) 0,
+				),
+				right-rail-advanced-main: (
+					gap: var(--global-spacing-6),
+					max-width: calc(var(--content-max-width) * 1px),
+					width: var(--content-scale-width),
+					margin: auto,
+				),
+				right-rail-advanced-full-width-2: (
+					margin: var(--global-spacing-6) 0 0 0,
+				),
+				right-rail-advanced-rail-container: (
+					grid-template-areas: "main-side-rail-top" "main-interior-item-1" "main-side-rail-middle"
+						"main-interior-item-2" "main-side-rail-bottom",
+					grid-template-columns: initial,
+					grid-template-rows: 1fr,
+					grid-auto-rows: auto,
+					gap: var(--global-spacing-6),
+				),
+				right-rail-advanced-main-side-rail: (
+					padding: initial,
+					gap: var(--global-spacing-5),
+					display: contents,
+				),
+				right-rail-advanced-main-interior-item: (
+					padding: initial,
+					border-right: initial,
+					gap: var(--global-spacing-5),
+					display: contents,
+				),
+				right-rail-advanced-footer: (
+					margin: var(--global-spacing-10) 0 var(--global-spacing-8) 0,
+				),
+				right-rail-advanced-main-interior-item-1: (
+					grid-area: main-interior-item-1,
+				),
+				right-rail-advanced-main-interior-item-2: (
+					grid-area: main-interior-item-2,
+				),
+				right-rail-advanced-main-side-rail-top: (
+					grid-area: main-side-rail-top,
+				),
+				right-rail-advanced-main-side-rail-middle: (
+					grid-area: main-side-rail-middle,
+				),
+				right-rail-advanced-main-side-rail-bottom: (
+					grid-area: main-side-rail-bottom,
+				),
+			),
+		),
+		desktop: (
+			blocks: (
+				right-rail-navigation: (
+					padding: 0 0 var(--global-spacing-8) 0,
+				),
+				right-rail-rail-container: (
+					grid-template-columns: 2fr 1fr,
+					gap: initial,
+					grid-template-rows: initial,
+				),
+				right-rail-main-side-rail: (
+					padding: 0 0 0 var(--global-spacing-6),
+				),
+				right-rail-main-interior-item: (
+					border-right: 1px solid #dadada,
```

Comment:
This hex value isn't one of our neutral color tokens, so assuming that using the hex is fine.

Reply:
yeah the color wasn't available to one of the neutrals. I can add it to the design inconsistencies list. I don't think this should be a blocker for achieving parity.

```
		"neutral-1": #e6efff,
		"neutral-2": #c3cfe3,
		"neutral-3": #a3b0c6,
		"neutral-4": #8593aa,
		"neutral-5": #69778e,
		"neutral-6": #4f5c71,
		"neutral-7": #384355,
		"neutral-8": #232b39,
		"neutral-9": #10151c,
		"neutral-10": #000000,